app/views/provider_interface/interviews/review_cancel.html.erb

```diff
@@ -0,0 +1,32 @@
+<% content_for :browser_title, "#{@application_choice.application_form.full_name} – #{@application_choice.course.name_and_code} - Review Interview cancellation reason" %>
+<% content_for :before_content, govuk_back_link_to(request.referer) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+    <h1 class="govuk-label-wrapper">
+      <label class="govuk-label govuk-label--l">
+        <span class="govuk-caption-l">
+          <%= @interview.application_choice.application_form.full_name %>
+        </span>
+        Check and send interview cancellation
+      </label>
+    </h1>
+
+    <%= render SummaryCardComponent.new(editable: true, border: false, rows: [
+    {
+      key: 'Reason for cancelling interview',
+      value: @interview.cancellation_reason
+    },
+    {}
```

Comment:
What's this for?

Reply:
This was part of my PR, not sure why the commits are stil available here.
It's to add an extra line at the end of the SummaryCard component so it looks similar to the one in the prototype.

Reply:
OK — let's get a clean rebase on this branch! Then find a better way to achieve that